src/libraries/libraries-packages.proj

```diff
@@ -16,7 +16,8 @@
 
   <ItemGroup>
     <ProjectReference Include="$(PkgDir)*\*.proj" Exclude="$(PkgDir)test\*" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)*\pkg\**\*.pkgproj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)*\src\**\*.csproj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" />
```

Comment:
I believe @ericstj was suggesting to do `$(MSBuildThisFileDirectory)*\src\**\*.*proj` instead of having two entries, one for `csproj` and the other for `ilproj`.